docs/documentation/using-notify.md

```diff
@@ -84,10 +84,10 @@ example:
       <form class="form" method="post">
 
         <div class="govuk-form-group">
```

Comment:
Did we consider rewriting this to use Nunjucks instead of HTML?

Reply:
Not until now, but it's a great call